testing/btest/core/mmdb/lookup.zeek

```diff
@@ -0,0 +1,28 @@
+# @TEST-DOC: Test basic DB lookups for success/failure.
+#
+# @TEST-REQUIRES: grep -q "#define USE_GEOIP" $BUILD/zeek-config.h
```

Comment:
We have `zeekl-config --have-geoip` now, no? :-)

Reply:
Oh yeah totally. :) That spot is what originally prompted me to add it. But last night it was close to midnight and I wanted to ship this thing. Happy to work it over today.